src/main/webapp/js/common/react/tests/Navbar.test.js

```diff
@@ -14,14 +14,19 @@ const navbarUrls = [
     name: 'Projects',
     requiresAuth: false,
   },
+  {
```

Comment:
Thanks for adding this!

Reply:
Np! It's definitely better to have the test URLs match up with the production URLs anyways.